### PR TITLE
Add regression test info to README.md

### PR DESCRIPTION
**Pull request by jdavies-st**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,56 +18,83 @@ To install the latest released version of the library, we recommend using [conda
 and the [astroconda](https://astroconda.readthedocs.io) channel to install the
 software and using a conda environment for the installation:
 
-    conda create -n jwst --override-channels -c http://ssb.stsci.edu/astroconda-dev -c defaults python=3 jwst
-    source activate jwst
+    conda create -n jwst --override-channels -c http://ssb.stsci.edu/astroconda-dev -c defaults python=3.6 jwst
+    conda activate jwst
```
Comment (hbushouse) [inline README.md]: Since when did the "source" command change to "conda"? Do both still work?

File: README.md [hunk 2]
```diff
@@ -18,56 +18,83 @@ To install the latest released version of the library, we recommend using [conda
...
 Regression Tests
 ----------------
 
-Latest regression test results can be found here:
+Latest regression test results can be found here (internal STScI only):
 
 https://boyle.stsci.edu:8081/job/RT/job/JWST/
 
-Test builds start at 10am and 7pm local time every day.
+The test builds start at 10am and 7pm local time every day on jwcalibdev.
```
Comment (hbushouse) [inline README.md]: perhaps "local **Baltimore** time"

File: README.md (hunk 1 above)
Comment (jdavies-st) [reply README.md]: It changed in the last few months.  I believe both still work.  If you have python 3.6 and the latest conda though, `conda` works.  I can back out this change if you prefer.

Comment (jdavies-st) [reply README.md]: Ah, ok.  I will change it back.

**Comment by jdavies-st** [comment]
Otherwise are we good to merge?

**Comment by stscicrawford** [comment]
I had two comment that I forgot to submit -- let me finish the review

File: README.md [hunk 3]
```diff
@@ -18,56 +18,83 @@ To install the latest released version of the library, we recommend using [conda
...
+
+
+Unit Tests
+----------
+
+Unit tests can be run via `pytest`.  We recommend using `pytest-xdist` so you can run them in parallel.  Install `pytest-xdist` and run pytest in the top level of the repository
+
+    conda install pytest-xdist
+    pytest -n N
+
```
Comment (stscicrawford) [inline README.md]: Maybe use consistent way through the document to specify option arguments (either <> or [])

**Pull request merged**